src/Sulu/Component/Rest/ListBuilder/Filter/DropdownFilterType.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\Rest\ListBuilder\Filter;
+
+use Sulu\Component\Rest\ListBuilder\FieldDescriptorInterface;
+use Sulu\Component\Rest\ListBuilder\ListBuilderInterface;
+
+class DropdownFilterType implements FilterTypeInterface
+{
+    public function filter(
+        ListBuilderInterface $listBuilder,
+        FieldDescriptorInterface $fieldDescriptor,
+        $options
+    ): void {
+        if (!is_string($options)) {
+            throw new InvalidFilterTypeOptionsException(
+                'The DropdownFilterType requires its options to be comma-separated list of values'
+            );
+        }
+
+        $listBuilder->in($fieldDescriptor, explode(',', $options));
```

Comment:
i think this will break if `$options`is an empty string

Reply:
It does not break, it just returns nothing :see_no_evil: Since somebody would to actually build send the request manually for this to happen, I would accept this for now.